docs/platforms.rst

```diff
@@ -0,0 +1,51 @@
+.. _platforms:
+
+Platforms
+=========
+
+Bottery design always considered the possibility to integrate to any chat platform. At the code level, there're many abstractions to make it possible. Bottery works out of the box with two chat platforms, Facebook Messenger and Telegram but it give the tools to easly integrate to any specific platform.
```

Comment:
I think that the first phrase could be rephrased to "Bottery has a simples way to integrate to a chat plaform, and it's design is made in a such a way that it could, eventually, integrate to any chat platform".

And the last phrase should start with a "For now, Bottery works with two chat platforms". (eliminate th "out of the box"). The last "but" could be removed because the ability to add any other platform was better described on the first phrase.